workflows/rnaseq/downstream/functional-enrichment.Rmd

```diff
@@ -31,7 +35,7 @@ deg.list <- nested.lapply(sel.list, rownames)
 genes.df <- nested.lapply(deg.list, bitr, fromType=from.type, toType=types, OrgDb=orgdb)
 ```
 
-```{r enrich_all, results='hide', cache=TRUE, dependson='final_clusters'}
+```{r enrich_all, results='hide'}
```

Comment:
no more caching?

Reply:
that was fine for the test data. Will have to test with regular size data set, and caching will be needed